### ISSUE TITLE
npm install and redoc-cli bundle fails

### ISSUE THREAD

**Issue by pohmelie** [Redocly/redoc#1442]
After last update to 0.10.0 `redoc-cli bundle` stop working.
How to reproduce:
## Start node latest image
``` 
$ docker run --rm -it node:latest bash
```
## Install redoc-cli globaly
``` 
root@002d9cd64e60:/# npm install -g redoc-cli
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: mobx@6.0.1
npm WARN node_modules/redoc-cli/node_modules/mobx
npm WARN   mobx@"^6.0.1" from redoc-cli@0.10.0
npm WARN   node_modules/redoc-cli
npm WARN     redoc-cli@"*" from the root project
npm WARN   1 more (redoc)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer mobx@"^5.15.4 || ^4.15.4" from mobx-react@6.3.1
npm WARN node_modules/redoc-cli/node_modules/mobx-react
npm WARN   mobx-react@"^6.2.2" from redoc@2.0.0-rc.45
npm WARN   node_modules/redoc-cli/node_modules/redoc

added 265 packages, and audited 265 packages in 12s

16 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice 
npm notice New patch version of npm available! 7.0.3 -> 7.0.6
npm notice Changelog: https://github.com/npm/cli/releases/tag/v7.0.6
npm notice Run npm install -g npm@7.0.6 to update!
npm notice 
```
## Try to bundle something
``` 
root@002d9cd64e60:/# redoc-cli bundle /path/to/openapi3.yml
/usr/local/lib/node_modules/redoc-cli/node_modules/mobx-react/node_modules/mobx/lib/mobx.js:20
        throw new Error("[mobx] " + (message || OBFUSCATED_ERROR));
        ^

Error: [mobx] There are multiple, different versions of MobX active. Make sure MobX is loaded only once or use `configure({ isolateGlobalState: true })`
    at invariant (/usr/local/lib/node_modules/redoc-cli/node_modules/mobx-react/node_modules/mobx/lib/mobx.js:20:15)
    at fail (/usr/local/lib/node_modules/redoc-cli/node_modules/mobx-react/node_modules/mobx/lib/mobx.js:15:5)
    at Timeout._onTimeout (/usr/local/lib/node_modules/redoc-cli/node_modules/mobx-react/node_modules/mobx/lib/mobx.js:1487:17)
    at listOnTimeout (node:internal/timers:555:17)
    at processTimers (node:internal/timers:498:7)
root@002d9cd64e60:/# 
```

**Comment by RomanHotsiy** [comment]
This is caused by new npm@7 (shipped with nod@15) is installing peer dependencies by default and a dependency we use (mobx-react) specifies the wrong peer dependency: 

`mobx@"^5.15.4 || ^4.15.4" from mobx-react@6.3.1`

As a workaround use the LTS version of node.js.

I will open a pr aginst mobx-react later.

**Comment by sergle** [comment]
Using LTS node still cause bundle errors:

```
docker run --rm -ti node:lts-buster-slim bash
root@7063a1088edd:/# yarn global add redoc-cli
yarn global v1.22.5
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "redoc-cli > redoc@2.0.0-rc.45" has unmet peer dependency "core-js@^3.1.4".
warning "redoc-cli > styled-components@5.2.1" has unmet peer dependency "react-is@>= 16.8.0".
warning "redoc-cli > redoc > mobx-react@6.3.1" has incorrect peer dependency "mobx@^5.15.4 || ^4.15.4".
warning "redoc-cli > redoc > mobx-react > mobx-react-lite@2.2.2" has incorrect peer dependency "mobx@^4.0.0 || ^5.0.0".
warning "redoc-cli > redoc > swagger2openapi > better-ajv-errors@0.6.7" has unmet peer dependency "ajv@4.11.8 - 6".
[4/4] Building fresh packages...
success Installed "redoc-cli@0.9.13" with binaries:
      - redoc-cli
Done in 13.59s.
```

```
root@7063a1088edd:/# redoc-cli bundle /tmp/openapi.yaml 
Prerendering docs
Error: Objects are not valid as a React child (found: object with keys {[object Object]}). If you meant to render a collection of children, use an array instead.
    in span
    in h1
    in Styled(styled.h1)
    in div
    in styled.div
    in div
    in styled.div
    in div
    in styled.div
    in ApiInfo
    in div
    in styled.div
    in div
    in styled.div
    in Context.Provider
    in Context.Provider
    in Context.Provider
    in Unknown
    in Redoc
    in Context.Provider
    in Context.Provider
    in ae
    at traverseAllChildrenImpl (/usr/local/share/.config/yarn/global/node_modules/react/cjs/react.development.js:1061:17)
    at traverseAllChildrenImpl (/usr/local/share/.config/yarn/global/node_modules/react/cjs/react.development.js:1023:23)
    at traverseAllChildren (/usr/local/share/.config/yarn/global/node_modules/react/cjs/react.development.js:1092:10)
    at mapIntoWithKeyPrefixInternal (/usr/local/share/.config/yarn/global/node_modules/react/cjs/react.development.js:1174:3)
    at toArray (/usr/local/share/.config/yarn/global/node_modules/react/cjs/react.development.js:1227:3)
    at ReactDOMServerRenderer.renderDOM (/usr/local/share/.config/yarn/global/node_modules/react-dom/cjs/react-dom-server.node.development.js:3954:18)
    at ReactDOMServerRenderer.render (/usr/local/share/.config/yarn/global/node_modules/react-dom/cjs/react-dom-server.node.development.js:3484:21)
    at ReactDOMServerRenderer.read (/usr/local/share/.config/yarn/global/node_modules/react-dom/cjs/react-dom-server.node.development.js:3373:29)
    at Object.renderToString (/usr/local/share/.config/yarn/global/node_modules/react-dom/cjs/react-dom-server.node.development.js:3988:27)
    at /usr/local/share/.config/yarn/global/node_modules/redoc-cli/index.js:215:29

```

I used 0.9.12 before, but now it's broken
```
root@da59594ea70f:/# redoc-cli --version
0.9.12

root@da59594ea70f:/# redoc-cli bundle /tmp/openapi.yaml 
Prerendering docs
TypeError: Object(...) is not a function
    at new GroupModel (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:10182:49)
    at /usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:10292:29
    at Array.map (<anonymous>)
    at mapHeadingsDeep (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:10291:26)
    at Function.MenuBuilder.addMarkdownItems (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:10303:16)
    at Function.MenuBuilder.buildStructure (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:10270:45)
    at new SpecStore (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:10161:53)
    at new AppStore (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:12804:21)
    at Module.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:12788:43)
    at step (/usr/local/share/.config/yarn/global/node_modules/tslib/tslib.js:140:27)

```

**Comment by RomanHotsiy** [comment]
Fixed in redoc-cli@0.10.1
